Source/Core/VideoCommon/TextureDecoder_x64.cpp

```diff
@@ -311,8 +312,14 @@ static void TexDecoder_DecodeImpl_I4_SSSE3(u32* dst, const u8* src, int width, i
       }
     }
   }
-#endif
 }
+#else
+static void TexDecoder_DecodeImpl_I4_SSSE3(u32* dst, const u8* src, int width, int height,
```

Comment:
From IRC:
what do you think about always defining FUNCTION_TARGET_SSSE3 and moving the SUPPORTS_FUNCTION_TARGET_SSSE3 in the function body? So we don't have to define the empty declaration again